gatsby-browser.js

```diff
@@ -14,4 +14,4 @@ import 'gatsby-theme-patternfly-org/global.css';
 import './dist/patternfly.css';
 
 // Utilities
-import './dist/patternfly-addons.css';
```

Comment:
This is probably going to break more than just our docs site. I believe many products use our addons. Can we keep it at the root of `dist` when we build?

Reply:
I realize we want to minimize the level of breakage we cause. One thing to note is by making this change will cause the scss and css files do not live along side of each other. We could move both charts and addons back to the top level and remove the addons directory completely for now.